demos/app/data-operations/data-state-configurator.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, Input, ViewChild, OnInit, EventEmitter, Output, ChangeDetectionStrategy } from "@angular/core";
+import { DataContainer, DataUtil, DataState, DataType,
```

Comment:
import * as Infragistics from '../../src/main'; everywhere we are referencing classes, interfaces, components from src/main we should use the alias Infragistics(or whatever you are using).